<testsuites id="" name="" tests="3" failures="1" skipped="0" errors="0" time="9.303533999999999">
<testsuite name="signout.spec.ts" timestamp="2025-08-08T07:57:11.421Z" hostname="chromium" tests="3" failures="1" skipped="0" time="14.695" errors="0">
<testcase name="User can sign in and then sign out" classname="signout.spec.ts" time="3.163">
</testcase>
<testcase name="Session does not persist after sign out and reload" classname="signout.spec.ts" time="3.176">
</testcase>
<testcase name="Access to dashboard after logout is blocked" classname="signout.spec.ts" time="8.356">
<failure message="signout.spec.ts:37:7 Access to dashboard after logout is blocked" type="FAILURE">
<![CDATA[  [chromium] › signout.spec.ts:37:7 › Access to dashboard after logout is blocked ──────────────────

    Error: Timed out 5000ms waiting for expect(page).toHaveURL(expected)

    Expected pattern: /.*\/login/
    Received string:  "https://meal4v.vercel.app/dashboard"
    Call log:
      - Expect "toHaveURL" with timeout 5000ms
        9 × unexpected value "https://meal4v.vercel.app/dashboard"


      50 |   
      51 |     // ✅ Should redirect or show "not authorized"
    > 52 |     await expect(page).toHaveURL(/.*\/login/);
         |                        ^
      53 |     await expect(page.locator('[data-testid="login-button"]')).toBeVisible();
      54 |   });
      55 |   
        at /Users/sina/Desktop/wollie_software/MVP/tests/signout.spec.ts:52:24

    attachment #1: screenshot (image/png) ──────────────────────────────────────────────────────────
    ../test-results/signout-Access-to-dashboard-after-logout-is-blocked-chromium/test-failed-1.png
    ────────────────────────────────────────────────────────────────────────────────────────────────

    attachment #2: video (video/webm) ──────────────────────────────────────────────────────────────
    ../test-results/signout-Access-to-dashboard-after-logout-is-blocked-chromium/video.webm
    ────────────────────────────────────────────────────────────────────────────────────────────────

    Error Context: ../test-results/signout-Access-to-dashboard-after-logout-is-blocked-chromium/error-context.md
]]>
</failure>
<system-out>
<![CDATA[
[[ATTACHMENT|test-results/signout-Access-to-dashboard-after-logout-is-blocked-chromium/test-failed-1.png]]

[[ATTACHMENT|test-results/signout-Access-to-dashboard-after-logout-is-blocked-chromium/video.webm]]

[[ATTACHMENT|test-results/signout-Access-to-dashboard-after-logout-is-blocked-chromium/error-context.md]]
]]>
</system-out>
</testcase>
</testsuite>
</testsuites>